Library/RSBot.Core/Components/CollisionManager.cs

```diff
@@ -78,6 +78,8 @@ public static bool HasActiveMeshes
     /// </value>
     public static List<CalculatedCollisionMesh>? ActiveCollisionMeshes { get; private set; }
```

Comment:
Remove the ?. Null checking is disabled on the project. 
Also please change the CenterRegionId field to CenterRegion and change ushort to Region struct.

```cs
    public static ushort CenterRegionId { get; private set; }
    // to
    public static Region CenterRegion { get; private set; }
```

also you can change the all region ushort based structs to `Region`